pkg/process/checks/checks.go

```diff
@@ -107,7 +107,7 @@ func (p CombinedRunResult) RealtimePayloads() []model.MessageBody {
 // If you want to add a check you MUST register it here.
 func All(config, sysprobeYamlCfg ddconfig.ReaderWriter, syscfg *sysconfigtypes.Config) []Check {
 	return []Check{
-		NewProcessCheck(config),
+		NewProcessCheck(config, sysprobeYamlCfg),
```

Comment:
Was using the init function not an option for pulling the config values?

Reply:
I figured it would be more complicated to do so. Also, this is the exact way that the connections check passes this config down, so for consistency purposes this seemed the best.

Reply:
Why do you think its more complicated?

Reply:
I had a harder time following the the call hierarchy of the Init Calls, and it felt like the code change was going to be larger. Also, since this is exactly the way the connections check does it for the same purpose, I didnt see a reason to make the implementation differently.